### ISSUE TITLE
Visually hidden programmatic headings

### ISSUE THREAD

**Issue by kengdoj** [atbcb/ICTTestingBaseline#559]
Are offscreen programmatic headings a WCAG 1.3.1 failure? The programmatic structure differs from visual structure.

**Comment by kengdoj** [comment]
offscreen programmatic headings are NOT a 1.3.1 issue, but should be tested for 2.4.6. Update 13.A Identify Content to include non-empty programattic headings

**Comment by drewnielson** [comment]
Agreed. Thanks!
